harvest-refactor/helper.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+# Dictionary mapping institutions to sets that should not be harvested
+do_not_harvest = {
+                 "auc": ['p15795coll31', 'p15795coll32', 'p15795coll25', 'p15795coll29', 'p15795coll22', 'p15795coll7',
+                 		'p15795coll3', 'p15795coll20', 'p15795coll17', 'p15795coll18'],
+}
+
+# Common helper functions
+def get_values_if_any(d, key):
+    return d.get(key, [])
+
+def to_str(bytes_or_str):
+    '''Takes bytes or string and returns string'''
+    if isinstance(bytes_or_str, bytes):
+        value = bytes_or_str.decode('utf-8')
+    else:
+        value = bytes_or_str
+    return value  # Instance of str
+
+def write_records(records, directory):
+    record_number = 0
+    for record in records:
```

Comment:
Can use `for record_number, record in enumerate(records):` instead of separately keeping track of record number. Only thing to be aware of is enumerate is 0 based.